packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -500,6 +500,11 @@ - (void)captureOutput:(AVCaptureOutput *)output
       return;
     }
 
+    // ignore audio samples until the first video sample arrives to avoid black frames
+    if (_videoWriter.status != AVAssetWriterStatusWriting && output != _captureVideoOutput) {
+      return;
```

Comment:
In `FLTCam.m` it seems there is no way it could capture just audio. In the high level dart interface I see that just audio is optional with the `enableAudio` parameter in the `CameraController` constructor. Even if it recorded just audio it would probably result in black video with that audio as the output file format is always `AVFileTypeMPEG4` (this is just speculation but I never seen pure audio clip to be in mp4).